src/main/java/teammates/storage/api/StudentsDb.java

```diff
@@ -364,15 +372,25 @@ public void updateStudentWithoutSearchability(String courseId, String email,
             String newName,
             String newTeamName, String newSectionName, String newEmail,
             String newGoogleID,
+            String newComments,
+            boolean keepUpdateTimestamp) throws InvalidParametersException,
+            EntityDoesNotExistException {
+        updateStudent(courseId, email, newName, newTeamName, newSectionName,
+                                        newEmail, newGoogleID, newComments, false, keepUpdateTimestamp);
+    }
```

Comment:
The header comment needs to be updated to explain the meaning of last parameter.